guides/micronaut-cloud-trace-google/micronaut-cloud-trace-google.adoc

```diff
@@ -0,0 +1,100 @@
+external:micronaut-cloud-trace-base/start.adoc[]
+* A https://cloud.google.com/gcp/[Google Cloud Platform (GCP)] account and a GCP project.
+
+common:costs-gcloud.adoc[]
+
+common:gcp.adoc[]
+
+external:micronaut-cloud-trace-base/create-app.adoc[]
+
+=== Tracer Configuration
+
+Using Micronaut CLI or Launch to create your application, the OpenTelemetry exporter will be added to your `application.yml` configurations.
```

Comment:
I'm not completely sure I understand what this is supposed to say. Is the exporter added automatically, no matter what, or is it added automatically only if you use Micronaut CLI or Launch?

```suggestion
If you use Micronaut CLI or Launch to create your application, the OpenTelemetry exporter will automatically be added to your `application.yml` configurations.
```
If this is the case, are there other ways to create an application in which the exporter is NOT automatically added? Or should this read more like the following?

```suggestion
Use Micronaut CLI or Launch to create your application. You will see that the OpenTelemetry exporter is automatically added to your `application.yml` configurations.
```

Reply:
Fixed in https://github.com/micronaut-projects/micronaut-guides/pull/964